contrib/completion/bash/docker

```diff
@@ -3035,7 +3035,8 @@ _docker_service_update_and_create() {
 
 	if [ "$subcommand" = "create" ] ; then
 		options_with_args="$options_with_args
-			--constraint
+			--constraint-add
+			--constraint-rm
```

Comment:
This doesn't look right, because `docker service create` only has `--constraint`, not `--constraint-add` / `--constraint-rm`. The completion for that is done in the `if [ "$subcommand" = "update" ]` section; https://github.com/docker/docker/blob/1ecaed0a99331ff6140e0a714e1b50d466597775/contrib/completion/bash/docker#L3085-L3089